fpsim/locations/kenya/kenya.py

```diff
@@ -51,435 +36,25 @@ def filenames():
     files['methods'] = 'mix.csv'
```

Comment:
@emilydriano can you help me with a question? I see that mix.csv is called here, which looks like it's just an array with a percentage for each method. We should be using method_mix_matrix_switch.csv, which has different arrays depending on postpartum, previous method, and age. Is that implemented somewhere else and I'm missing it?? I know in a previous version the percentages ere hard coded rather than reading in the csv, so maybe that's happening somewhere and I can't find it?

Reply:
@mzimmermann-IDM yes, so method_mix_matrix_switch.csv is indeed being used but isn't referenced directly in kenya.py... it's used in the function process_markovian_method_choice (which is in data_utils.py) which is then called in init_method_pars in methods.py. The file mix.csv is only being used for calibration (comparing the method mix at the end of the sim to the values in the 'mix' file).

Reply:
Ok great, thanks @emilydriano ! I was looking at the relevant to tutorial 3 where you can update the method mix. In the intervention, method mix is replaces with a single array so I was trying to figure out how it works. Does it replace all of the different arrays in the matrices from the csv with the user defined array? I think that must be what's happening but can you confirm? 

The other thing I can't find explicitly documented is the order of the methods. I assume it is alphabetical as it is in the csv's, but would be great to confirm (and include very clearly in all the documentation @MObrien-IDM )

Reply:
Good question.. so this intervention sets the probabilities of methods being selected by modifying the contraceptive module's pars['method_mix'], which normally by default sets all methods to having equal probability of being chosen (i.e. with 9 methods, each has prob of ~0.111). But it looks like this param is only being used in the RandomChoice module and not the Simple or Standard choice ones.. @robynstuart can you confirm? 
![image](https://github.com/user-attachments/assets/e9179309-2826-46a5-9e52-fe81f07951b5)

The method mix order is the same as those in the 'method_list' in methods.py, starting with 'pill': 
![image](https://github.com/user-attachments/assets/39b8f786-c53f-47a7-9e2e-0f263110d873)